urlchecker/client/check.py

```diff
@@ -27,6 +27,18 @@ def main(args, extra):
       - args: the argparse ArgParser with parsed args
       - extra: extra arguments not handled by the parser
     """
+    if args.command == "check_ci":
```

Comment:
This could be implemented just by defining a variable for an explicit filename. I'll also ask again - why can't we just give the name of a file to the file patterns? The functionality would be equivalent. I'm not totally clear on what you are trying to do.

Reply:
> I'll also ask again - why can't we just give the name of a file to the file patterns? The functionality would be equivalent.

I'm doing this now.